app/views/attendances/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(@attendance) do |f| %>
+  <% if @attendance.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@attendance.errors.count, "error") %> prohibited this attendance from being saved:</h2>
+
+      <ul>
+      <% @attendance.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= f.label :date %><br>
+    <%= f.text_field :date %>
+    <%= f.label :present %><br>
+    <!-- #Don't know what to put here -->
```

Comment:
Are you suggesting at the model level? 
 class Attendance <ActiveRecord::Base
    has_many :sessions
    has_many :students, :through => :course_rosters

or just in the views?